src/banking_stage.rs

```diff
@@ -99,14 +120,61 @@ impl BankingStage {
                     },
                 }).collect();
 
-            debug!("process_transactions");
-            let results = bank.process_transactions(transactions);
-            let transactions = results.into_iter().filter_map(|x| x.ok()).collect();
-            if let Err(_) = signal_sender.send(Signal::Transactions(transactions)) {
-                return Err(Error::SendError);
+            let mut chunk_start = 0;
+            while chunk_start != transactions.len() {
```

Comment:
this doesn't yet compile, so tests aren't here yet.  I'm looking for feedback on how better to break this up